src/style.css

```diff
@@ -0,0 +1,121 @@
+﻿:root {
+  --color-light-blue: #00acdc;
+  --text-color: #060b35;
+}
+
+* {
+  margin: 0;
+
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.product-card__img {
+  display: block;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.product-card__description {
+  margin: 0 0 4px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+
+  color: var(--text-color);
+}
+
+.product-card__code {
+  color: #616070;
```

Comment:
if u use color variables, better use it for all colors

Reply:
Fixed. In my opinion, variables are needed if the color is used more than once.